src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/mensageria/Publisher.java

```diff
@@ -0,0 +1,18 @@
+package com.projetogerenciamentocurso.gerenciamentocurso.mensageria;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.stereotype.Component;
+
+import lombok.AllArgsConstructor;
+@Component
```

Comment:
Correção de espaçamento para separar a classe das importações
```suggestion
import lombok.AllArgsConstructor;

@Component
```

Reply:
feito